src/Microsoft.Security.Utilities.Core/PotentialSecurityKeys/SEC000_000_Unclassified32ByteBase64String.cs

```diff
@@ -38,14 +38,6 @@ public override IEnumerable<Detection> GetDetections(string input,
         {
             string match = input.Substring(detection.Start, detection.Length);
 
-            if (!object.Equals(azure32ByteIdentifiableKeys.GetDetections(match,
-                                                                         generateSha256Hashes,
-                                                                         defaultRedactionToken,
-                                                                         regexEngine).FirstOrDefault(), objB: default))
-            {
-                continue;
-            }
-
```

Comment:
Hmm, I don't see this mentioned in PR description or release notes.

We're giving up on avoiding double-firing in a cumbersome way? This was actually my motivating example in #199.

If this is intended to stay out, then I think the whole override can be removed and Azure32ByteIdentifiableKeys can be deleted.

(Ditto all of the above for Azure64Byte.)

Reply:
No, sorry, this is a good catch, bad code removal! reverting